### ISSUE TITLE
Node v16.14.0 Missing from rules_nodejs v4.6.3 

### ISSUE THREAD

**Issue by twheys** [bazelbuild/rules_nodejs#3345]
I am trying to set the new LTS version of node, v16.14.0, as my node version with rules nodejs v4.6.3. It appears that that version of node was added but perhaps missed in the 4.x branch. Furthermore, setting my own version parameters in the `node_repositories` workspace rule does not seem to get around the missing version because the code checking if a node version exists relies purely on the published list of node versions (see https://github.com/bazelbuild/rules_nodejs/blob/stable/nodejs/private/os_name.bzl)

```
node_repositories(
    node_repositories = {
        "16.14.0-darwin_arm64": ("node-v16.14.0-darwin-arm64.tar.gz", "node-v16.14.0-darwin-arm64", "56e547d22bc7be8aa40c8cfd604c156a5bcf8692f643ec1801c1fa2390498542"),
        "16.14.0-darwin_amd64": ("node-v16.14.0-darwin-x64.tar.gz", "node-v16.14.0-darwin-x64", "26702ab17903ad1ea4e13133bd423c1176db3384b8cf08559d385817c9ca58dc"),
        "16.14.0-linux_arm64": ("node-v16.14.0-linux-arm64.tar.xz", "node-v16.14.0-linux-arm64", "5a6e818c302527a4b1cdf61d3188408c8a3e4a1bbca1e3f836c93ea8469826ce"),
        "16.14.0-linux_ppc64le": ("node-v16.14.0-linux-ppc64le.tar.xz", "node-v16.14.0-linux-ppc64le", "d1012696cacb3833b8b33748905e716f2524766b29a2a4c405f34ed2f3e5fdb4"),
        "16.14.0-linux_s390x": ("node-v16.14.0-linux-s390x.tar.xz", "node-v16.14.0-linux-s390x", "1c98494bc097547bcadb972780aec58bb40b9c094f0daed75debfee4cb980fd9"),
        "16.14.0-linux_amd64": ("node-v16.14.0-linux-x64.tar.xz", "node-v16.14.0-linux-x64", "0570b9354959f651b814e56a4ce98d4a067bf2385b9a0e6be075739bc65b0fae"),
        "16.14.0-windows_amd64": ("node-v16.14.0-win-x64.zip", "node-v16.14.0-win-x64", "c783f32aa22758e9fdcabb23daf348cc52f876fbd679d54edc2c4921ccd6d6c5"),
    },
    node_urls = ["https://nodejs.org/dist/v{version}/{filename}"],
    node_version = "16.14.0",
)
```

**Comment by mattem** [comment]
This was fixed in #3342, which went out in [4.6.3](https://github.com/bazelbuild/rules_nodejs/releases/tag/4.6.3).

This minimal repro works as expected here:

```
# WORKSPACE
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")

http_archive(
    name = "build_bazel_rules_nodejs",
    sha256 = "ad3e5afa52ef9aac4da426f61e339c054ecbc0e6665cec2109f8846b4c8339e3",
    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/4.6.3/rules_nodejs-4.6.3.tar.gz"],
)

load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")

node_repositories(
    node_repositories = {
        "16.14.0-darwin_arm64": ("node-v16.14.0-darwin-arm64.tar.gz", "node-v16.14.0-darwin-arm64", "56e547d22bc7be8aa40c8cfd604c156a5bcf8692f643ec1801c1fa2390498542"),
        "16.14.0-darwin_amd64": ("node-v16.14.0-darwin-x64.tar.gz", "node-v16.14.0-darwin-x64", "26702ab17903ad1ea4e13133bd423c1176db3384b8cf08559d385817c9ca58dc"),
        "16.14.0-linux_arm64": ("node-v16.14.0-linux-arm64.tar.xz", "node-v16.14.0-linux-arm64", "5a6e818c302527a4b1cdf61d3188408c8a3e4a1bbca1e3f836c93ea8469826ce"),
        "16.14.0-linux_ppc64le": ("node-v16.14.0-linux-ppc64le.tar.xz", "node-v16.14.0-linux-ppc64le", "d1012696cacb3833b8b33748905e716f2524766b29a2a4c405f34ed2f3e5fdb4"),
        "16.14.0-linux_s390x": ("node-v16.14.0-linux-s390x.tar.xz", "node-v16.14.0-linux-s390x", "1c98494bc097547bcadb972780aec58bb40b9c094f0daed75debfee4cb980fd9"),
        "16.14.0-linux_amd64": ("node-v16.14.0-linux-x64.tar.xz", "node-v16.14.0-linux-x64", "0570b9354959f651b814e56a4ce98d4a067bf2385b9a0e6be075739bc65b0fae"),
        "16.14.0-windows_amd64": ("node-v16.14.0-win-x64.zip", "node-v16.14.0-win-x64", "c783f32aa22758e9fdcabb23daf348cc52f876fbd679d54edc2c4921ccd6d6c5"),
    },
    node_version = "16.14.0",
)
```

```
bazel run @nodejs//:node -- --version
INFO: Analyzed target @nodejs//:node (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target @nodejs_darwin_arm64//:bin/node up-to-date (nothing to build)
INFO: Elapsed time: 0.102s, Critical Path: 0.00s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Build completed successfully, 1 total action
v16.14.0
```

Can you check that you are definitely on 4.6.3, and if so, provide a minimal repro demonstrating the issue?

**Comment by twheys** [comment]
Hi, thanks for the quick response. I tried it again today and it works now. I had upgraded to v4.6.3 when I first tried this, so not sure what went wrong. Thanks, closing the issue.
